.travis.yml

```diff
@@ -2,9 +2,10 @@ language: python
 sudo: false
 python:
     - 2.7
+    - 3.3
     - 3.4
     - 3.5
-    - 3.6-dev
+    - 3.6
     - pypy
```

Comment:
you are right should also include pypy3 and maybe other Python implementation, will try if it works and may add it.